benchmark/benchmark.go

```diff
@@ -193,6 +122,27 @@ func DoUnaryCall(tc testpb.TestServiceClient, reqSize, respSize int) {
 	}
 }
 
+//DoStreamingcall performs a streaming RPC with given stub and request and response size.client side
+func DoStreamingCall(tc testpb.TestServiceClient, reqSize, respSize int) {
+	pl := newPayload(testpb.PayloadType_COMPRESSABLE, reqSize)
+	req := &testpb.SimpleRequest{
+		ResponseType: pl.Type,
+		ResponseSize: int32(respSize),
+		Payload:      pl,
+	}
+	stream, err := tc.StreamingCall(context.Background())
```

Comment:
This means you create a new stream every time, which is incorrect. You probably can use sync.Once to init the stream here.